src/Symfony/Component/Config/composer.json

```diff
@@ -17,7 +17,8 @@
     ],
     "require": {
         "php": ">=5.5.9",
-        "symfony/filesystem": "~2.8|~3.0"
+        "symfony/filesystem": "~2.8|~3.0",
+        "symfony/dependency-injection": "~3.2"
```

Comment:
I put this in require because we use the "PriorityTaggedServiceTrait" from DependencyInjection [here](https://github.com/symfony/symfony/pull/21375/files#diff-15aadc4fecbfc153132047c9b590e0acR26).

I add it to make tests passes on PHP7.1 (travis job).

If it needs to be in require-dev even with this, I change it.